docs/topics/db/examples/one_to_one.txt

```diff
@@ -47,7 +47,8 @@ Create a couple of Places::
     >>> p2 = Place(name='Ace Hardware', address='1013 N. Ashland')
     >>> p2.save()
 
-Create a Restaurant. Pass the ID of the "parent" object as this object's ID::
+Create a Restaurant. Reference the "parent" object by passing it as the value 
+for the ``OneToOneField``::
```

Comment:
This sentence it totally outdated (see 04544c97b92b2de485c6de6675eb387fafb758c7), but we should keep the previous meaning, e.g.
```
Create a Restaurant. Pass the "parent" object as this object's primary key:
```
It's crucial for this example that `place` is the `Restaurant`'s primary key.